sdcm/utils/benchmarks.py

```diff
@@ -129,9 +173,27 @@ def _get_number_of_cores_on_node(self) -> int:
         core_num = int(regex_pattern.search(cat_result).groupdict()["core_num"])
         return core_num
 
-    @staticmethod
-    def run_fio():
-        ...
+    def run_fio(self):
+        run_cmd = """\
+            cd cassandra-fio
+            ./fio_runner.sh lcs
+            ./fio_runner.sh stcs
+            ls ./reports
+        """
+        cassandra_fio_run_result = self._remoter.sudo(shell_script_cmd(run_cmd), new_session=True)
+        if cassandra_fio_run_result.stderr:
+            LOGGER.warning("cassandra-fio benchmark ran with errors: \n%s", cassandra_fio_run_result.stderr)
+        LOGGER.info("cassandra-fio results:")
+        LOGGER.info("%s", cassandra_fio_run_result.stdout)
+        return ScyllaNodeBenchmarkSysbenchResult(cassandra_fio_run_result.stdout)
 
     def run_benchmarks(self):
         self.run_sysbench()
+        self.run_fio()
+
+
+if __name__ == "__main__":
+    so = Path("sysbench_out")
```

Comment:
Please, look at our doc located at [1].
According to it "so" is not good naming. But, moreover, it is used just once, so it could be used just in place.

[1] https://docs.google.com/document/d/1jihgOKb5iGRlD8_HQ92O0JbLk1kASUoZT23i_MXFSKI/edit#heading=h.6byizauykwn9

Reply:
Fixed with rebase